### ISSUE TITLE
Default command won't execute on connect

### ISSUE THREAD

**Issue by jvalencia80** [connectbot/connectbot#438]
It will be showed on screen and you just have to press Return to execute it.

Is this a bug?

**Comment by leonerd** [comment]
Put the linefeed in the command box. That's so you can have multiple commands.. or script some sort of menu system that doesn't want a linefeed at the end.

**Comment by jvalencia80** [comment]
I don't get it, that's not standard ssh behaviour and it's not stated anywhere that you require a linefeed at the end. I've tried other ssh clients and only ConnectBot do this.

So you're not using the built-in ssh command execution but just printing a command and let it run with a linefeed?

**Comment by leonerd** [comment]
Correct; this is implemented by injecting all the keystrokes into the terminal at the far end, rather than running it as a separate command. It means that the "script" can interact even with interactive programs on the far end, rather than simply executing them.

**Comment by jvalencia80** [comment]
Sorry for opening this again, but I have another issue with this.

So I added the return at the end and it worked as you said, but when my menu script exits, it doesn't close the connection.

In the standard way (ssh host command), it will disconnect when the command ends. 

So I added this: `myscript.sh && exit\n`

It worked, but after exit executes and disconnects, it still asks to keep the connection alive even when it's already disconnected. If I deny disconnection, it just keeps the connection alive but I can't do anything with it (apparently).

Is it a bug or a feature?
